src/ngraph/runtime/cpu/pass/cpu_layout.cpp

```diff
@@ -279,7 +280,13 @@ namespace ngraph
                     auto arg0_shape = node->get_input_shape(0);
                     auto arg1_shape = node->get_input_shape(1);
 
-                    if (default_weights_format)
+                    if (default_weights_format && use_bias)
+                    {
+                        arg1_shape =
+                            std::dynamic_pointer_cast<ngraph::op::GroupConvolutionBias>(node)
```

Comment:
If you use a dynamic_pointer_cast you need to check that the result is not nullptr or Klocwork is going to complain. If you are **sure** that this is the correct type then you can use static_pointer_cast and no check is required. This is true for other instances of dynamic_pointer_cast like line 291.